index.html

```diff
@@ -14,25 +16,40 @@
       </ul>
     </nav>
     <section id="About">
-      <h1>Girma Ebssa</h1>
+      <div class="Wrapper">
+        <h1>Girma Ebssa</h1>
+        <img class = "img-coding" src="image/code.jpg" alt="How Girma practice coding">
+        
+      </div>
       <h2>About</h2>
-      <p>Ebssa G who is a CodeTheDream student attending io-programing.</p>
+      <p>Girma Ebssa has completed basic courses in programing, javascript, html and css through RefCode. 
+        I have also completed bubble bootcamp course to develop a web using no code platform approach from RefCode by volunteers.
+         A Bubble Bootcamp is a dedicated program that teaches you how to build web apps using the powerful no-code platform called Bubble.
+          These bootcamps are designed to help you master Bubble, regardless of your starting point.
+           They cover various aspects of app development, and you’ll learn how to create web applications without writing code.
+             Additionally, I am currently enrolled in the Io-programing course at CodeTheDream. 
+             CodeTheDream offers free code school programs and paid apprenticeships, providing real-world experience in app development for a diverse range of clients.</p>
     </section>
     <section id="Experience">
       <h2>Experience</h2>
       <ul>
-        <li>Ethiopia</li>
+        <li>Metropolitan University</li>
+        <p>I was an instructor at Metropolitan University. The university is located on the east side of the capital Addis Ababa. I was teaching
+          computer assisted instructions for both degree and diploma graduates including advanced and basic research methodologies. I got my first degree
+           from the same university and started my professional career.
+
+        </p>
         <li>Seattle</li>
         <li>Atlanta</li>
-        <li>Atlanta</li>
       </ul>
+
     </section>
     <section id="Connect">
       <h2>Connect</h2>
       <ul>
-        <li><a href="https://www.linkedin.com/">Linkedin</a></li>
-        <li><a href="https://twitter.com/?lang=en">X</a></li>
-        <li><a href="https://github.com/?lang=en">GitHub</a></li>
+        <li><a href="https://www.linkedin.com/"> <img class="linkedin-logo" src="image/linkedin.png" alt="linkedin logo">Linkedin</a></li>
+        <li><a href="https://twitter.com/?lang=en"> <img class="twitter-logo" src="image/twitter.png" alt="old twitter logo">X</a></li>
+        <li><a href="https://github.com/?lang=en"> <img class="github-logo" src="image/github-logo.png" alt="github logo">GitHub</a></li>
```

Comment:
Nice job with these. You might also look in something like [Font Awesome](https://fontawesome.com/) for icons.